src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/MediaSection.js

```diff
@@ -0,0 +1,30 @@
+//@flow
+import React from 'react';
+import {Datagrid, DatagridStore} from 'sulu-admin-bundle/containers';
+
+type Props = {
+    adapters: Array<string>,
```

Comment:
Does this need to be configureable? I'd say no...

Reply:
Yes because we need to distinguish between `MediaSelectionAdapter` and `MediaOverviewAdapter`